app/src/importimageseqdialog.cpp

```diff
@@ -275,7 +272,7 @@ const PredefinedKeySetParams ImportImageSeqDialog::predefinedKeySetParams() cons
     dot = finalList[0].lastIndexOf(".");
 
     QStringList absolutePaths;
-    for (QString fileName : finalList) {
+    for (const QString &fileName : finalList) {
```

Comment:
```suggestion
    for (const QString &fileName : qAsCost(finalList)) {
```

By adding const here, you'll probably get an additional warning for 'Might detach container', as sch we use qAsConst(...) to make the list const also.

Reply:
this was a separate warning with many occurrences but not all of them went away by adding `qAsCost()` so I just skipped it entirely. thanks for the info.

Reply:
nope, the CI doesn't like that.
![image](https://github.com/user-attachments/assets/48b56a4d-a857-4e2c-a8c8-6de081331d07)

Reply:
My bad, I recalled that we used it elsewhere in the code but it doesn't seem so.. it's likely because of backward compatibility then.
Alright ignore those comments 😅